Makefile

```
@@ -1,27 +1,39 @@
 .PHONY: test test-report test-fix update-compatibility-patch
 
 PHP_74_OR_NEWER:=$(shell php -r "echo (int) version_compare(PHP_VERSION, '7.4', '>=');")
+PHP_80_OR_NEWER:=$(shell php -r "echo (int) version_compare(PHP_VERSION, '8.0', '>=');")
 
 test: test-report test-fix
 
 test-report: vendor
 	@if [ $(PHP_74_OR_NEWER) -eq 1 ]; then git apply tests/php-compatibility.patch; fi
-	@vendor/bin/phpcs `find tests/input/* | sort` --report=summary --report-file=phpcs.log; diff -u tests/expected_report.txt phpcs.log; if [ $$? -ne 0 ]; then if [ $(PHP_74_OR_NEWER) -eq 1 ]; then git apply -R tests/php-compatibility.patch; fi; exit 1; fi
+	@if [ $(PHP_80_OR_NEWER) -eq 1 ]; then git apply tests/php-compatibility-80.patch; fi
+	@vendor/bin/phpcs `find tests/input/* | sort` --report=summary --report-file=phpcs.log; diff -u tests/expected_report.txt phpcs.log; if [ $$? -ne 0 ]; then if [ $(PHP_74_OR_NEWER) -eq 1 ]; then git apply -R tests/php-compatibility.patch; fi; if [ $(PHP_80_OR_NEWER) -eq 1 ]; then git apply -R tests/php-compatibility-80.patch; fi; exit 1; fi
 	@if [ $(PHP_74_OR_NEWER) -eq 1 ]; then git apply -R tests/php-compatibility.patch; fi
+	@if [ $(PHP_80_OR_NEWER) -eq 1 ]; then git apply -R tests/php-compatibility-80.patch; fi
```

The tests are failing because it is applying both the 7.4 and the 8.0 patches. 


My suggestion is to rename `tests/php-compatibility.patch` to `tests/php-compatibility-74.patch` and do the following:

```suggestion
	@if [ -f "$PATCH_FILE" ]; then git apply $PATCH_FILE ; fi
	@vendor/bin/phpcs `find tests/input/* | sort` --report=summary --report-file=phpcs.log; diff -u tests/expected_report.txt phpcs.log; if [ $$? -ne 0 ]; then if [ -f "$PATCH_FILE" ]; then git apply -R $PATCH_FILE; fi; exit 1; fi
	@if [ -f "$PATCH_FILE" ]; then git apply -R $PATCH_FILE ; fi
```


**Note:** `test-fix` and `update-compatibility-patch` would need updated as well to use `$PATCH_FILE`